dev/Debug.php

```diff
@@ -41,6 +41,17 @@ class Debug {
 	private static $friendly_error_header = 'There has been an error';
 
 	/**
+	 * Set to true to enable friendly errors to set a http response code corresponding to the error.
+	 * If left false then error pages will be served as HTTP 200.
+	 *
+	 * Will be removed in 4.0, and fixed to on.
+	 *
+	 * @config
+	 * @var bool
+	 */
+	private static $friendly_error_httpcode = false;
```

Comment:
The risk of regressions is extremely high were it to be fixed to true already. It could potentially interfere with a lot of existing infrastructural, so a phased approach (opt in by setting a config) is likely the least risky.

@hafriedlander can you please weigh in on this?